packages/framework/attributor/package.json

```diff
@@ -89,9 +71,11 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha --ignore \"dist/test/memory/**/*\" --recursive \"dist/test/**/*.spec.*js\" -r node_modules/@fluid-internal/mocha-test-setup",
```

Comment:
test/memory ignore is being removed. I don't see any files in such a path. Unused ignore being cleaned up?